packages/backend/src/database/migrations/20220727122405_drop_issuer_fk.ts

```diff
@@ -0,0 +1,31 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    if (await knex.schema.hasTable('openid_identities')) {
+        await knex.schema.alterTable('openid_identities', (tableBuilder) => {
+            tableBuilder.dropForeign('issuer');
```

Comment:
Will there be another way to know what rows are for google and for okta ? I think it would be useful 🤔

Reply:
We can still use `issuer`. The `dropForeign` command just drops the foreign constraint but leaves the column there

Reply:
But isn't the issuer unique for each okta org ?

Reply:
Yes that's true! I was thinking of pattern matching to domains that say `okta.com` but maybe that's a bit fragile